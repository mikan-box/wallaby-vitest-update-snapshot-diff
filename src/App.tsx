import * as Styles from "./App.styles";

export const App = () => {
  return <Styles.Wrapper>Wrapper</Styles.Wrapper>;
};
