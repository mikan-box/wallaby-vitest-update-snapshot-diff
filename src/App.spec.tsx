import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";

import { App } from "./App";

describe("App", () => {
  it("should render", () => {
    const { asFragment } = render(<App />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
