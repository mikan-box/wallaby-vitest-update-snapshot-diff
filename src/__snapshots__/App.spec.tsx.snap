// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > should render 1`] = `
<DocumentFragment>
  <div
    class="css-1npohkm-Wrapper euq26r80"
  >
    Wrapper
  </div>
</DocumentFragment>
`;
